jest.config.js

```diff
@@ -1,22 +1,19 @@
 module.exports = {
   preset: 'react-native',
+  testMatch: [
+    '<rootDir>/autolink/**/*.test.{js,ts}',
+    '<rootDir>/integration/**/*.test.{js,ts}',
+    '<rootDir>/lib/**/*.test.{js,ts}',
+    '<rootDir>/playground/**/*.test.{js,ts}',
+    '<rootDir>/scripts/**/*.test.{js,ts}',
+  ],
   transformIgnorePatterns: [
     'node_modules/(?!(@react-native|react-native|react-native-ui-lib|react-native-animatable|react-native-reanimated)/)',
   ],
   transform: {
     '\\.[jt]sx?$': 'babel-jest',
   },
-  roots: [
```

Comment:
Until latest detox version, detox didn't use jest.config.js, we use it for headless tests